crates/ruff/tests/version.rs

```diff
@@ -0,0 +1,105 @@
+//! Tests the --version command
+use std::fs;
+use std::process::Command;
+
+use anyhow::Result;
+use insta_cmd::{assert_cmd_snapshot, get_cargo_bin};
+use tempfile::TempDir;
+
+const BIN_NAME: &str = "ruff";
+const VERSION_FILTER: [(&str, &str); 1] = [(
+    r"\d+\.\d+\.\d+(\+\d+ \(\w{9} \d\d\d\d-\d\d-\d\d\))?",
```

Comment:
Possibly this regex is a little overboard 😆 I'd be okay with going with something a little more vague if you prefer, e.g.

```suggestion
    r"\d+\.\d+\.\d+.*",
```

Reply:
I don't have a strong preference. I prefer simple regex because I can't read regex expressions but 🤷